zipline/data/_array_utilities.pyx

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+cimport numpy as np
+cimport cython
+
+from bcolz.carray_ext cimport carray
+
+cdef inline int int_min(int a, int b): return a if a <= b else b
+
+@cython.cdivision(True)
+def minute_value(np.ndarray[np.long_t, ndim=1] arr, Py_ssize_t pos):
+    cdef short q, r
+
+    q = cython.cdiv(pos, 390)
+    r = cython.cmod(pos, 390)
+
+    return arr[q] + r
+
+def find_position_of_minute(np.ndarray[np.long_t, ndim=1] arr,
+                             np.long_t minute_val):
```

Comment:
I believe the convention we have been following is to import all numpy functions.
In part because that allows us to be specific about which are cimports or not.